packages/asset-swapper/CHANGELOG.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "5.4.0",
+        "changes": [
+            {
+                "note": "Add `takerAssetToEthRate` and `makerAssetToEthRate` to swap quote response",
+                "pr": 49
+            }
+        ],
+        "timestamp": 1607347477
```

Comment:
Did you add this timestamp manually or...?

Asking because the publish workflow works by looking for changelog entries WITHOUT timestamps, publishing them, and then adding a timestamp after publishing.

Reply:
I most certainly did... Removing it 🤦